### PR TITLE
Kotlin welcome email example

### PR DESCRIPTION
**Pull request by christyjacob4**

### CODE REVIEW

File: kotlin/welcome-email/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# 📧 Sending Welcome Emails using Mailgun's Email API
+A sample Kotlin Cloud Function for sending a welcome email to a newly registered user.
```
Comment (lohanidamodar) [inline kotlin/welcome-email/README.md:2]: For consistency new line after heading seems good

File: kotlin/welcome-email/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+## 📝 Environment Variables
```
Comment (lohanidamodar) [inline kotlin/welcome-email/README.md:4]: newline after headings

File: kotlin/welcome-email/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+* **MAILGUN_API_KEY** - API Key for Mailgun 
+* **MAILGUN_DOMAIN** - Domain Name from Mailgun
+
+## 🚀 Building and Packaging
+
+To package this example as a cloud function, follow these steps.
+
+* Import the project into IntelliJ, Eclipse or any other IDE that has support for Kotlin projects. 
+
+* Build a jar for the project. Here's a [Stack Overflow answer](https://stackoverflow.com/questions/1082580/how-to-build-jars-from-intellij-properly) to help you.
```
Comment (lohanidamodar) [inline kotlin/welcome-email/README.md:16]: Isn't there simple steps?

**Review by lohanidamodar — CHANGES_REQUESTED** [review]
Some white space issues. Rest LGTM

File: kotlin/welcome-email/README.md (hunk 3 above)
Comment (christyjacob4) [reply kotlin/welcome-email/README.md:16]: This is the way to configure and build jars.
I have explained the steps in the written tutorial. I just wanted to keep the README concise.

File: kotlin/welcome-email/README.md (hunk 1 above)
Comment (christyjacob4) [reply kotlin/welcome-email/README.md:2]: Done :+1:

File: kotlin/welcome-email/README.md (hunk 2 above)
Comment (christyjacob4) [reply kotlin/welcome-email/README.md:4]: Done :+1:

File: kotlin/welcome-email/README.md (hunk 3 above)
Comment (lohanidamodar) [reply kotlin/welcome-email/README.md:18]: Ok

**Review by lohanidamodar — APPROVED** [review]

**Review by abnegate — APPROVED** [review]

**Pull request merged**